docs/FAQ.md

```diff
@@ -0,0 +1,91 @@
+# Frequently Asked Questions
+
+
+### Scripting Runtime Environment not setup correctly
+
+If you haven't switched your scripting runtime version from .NET 3.5 to .NET 4.6, you will see such error message:
+
+```
+Assets/ML-Agents/Scripts/ExternalCommunicator.cs(259,24): error CS1061: Type `System.Text.StringBuilder' does not contain a definition for `Clear' and no extension method `Clear' of type `System.Text.StringBuilder' could be found. Are you missing an assembly reference?
+```
+
+This is because .NET 3.5 doesn't support method Clear() for StringBuilder, refer to [Setting Up ML-Agents Within Unity](Installation.md#setting-up-ml-agent-within-unity) for solution. 
+
+### TensorFlowSharp flag not turned on. 
+
+If you have already imported the TensorFlowSharp plugin, but havn't set ENABLE_TENSORFLOW flag for your scripting define symbols, you will see the following error message:
+
+```
+You need to install and enable the TensorFlowSharp plugin in order to use the internal brain. 
+```
+
+This error message occurs because the TensorFlowSharp plugin won't be usage without the ENABLE_TENSORFLOW flag, refer to [Setting Up ML-Agents Within Unity](Installation.md#setting-up-ml-agent-within-unity) for solution. 
+
+### Tensorflow epsilon placeholder error
+
+If you have a graph placeholder set in the internal Brain inspector that is not present in the TensorFlow graph, you will see some error like this:
+
+```
+UnityAgentsException: One of the Tensorflow placeholder could not be found. In brain Ball3DBrain, there are no FloatingPoint placeholder named epsilon. 
+```
```

Comment:
This is too specific. There should no longer be any epsilon placeholder in the repo

Reply:
Maybe replace this with

"UnityAgentsException: One of the Tensorflow placeholder could not be found. In brain <some_brain_name>, there are no FloatingPoint placeholder named <some_name>."